packages/components/src/__future__/Tag/RemoveableTag.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { HTMLAttributes } from "react"
```

Comment:
Export this file to `Tag/index.ts`

Reply:
Thanks, but I'd appreciate it if you didn't phrase all of your PR feedback as orders. It's quite degrading.